### PR TITLE
Include cause in ConfigurationException

### PR DESCRIPTION
**Pull request by brianguertin**
So, for debugging purposes, I would prefer to know the cause of this exception when there is one, and I don't see any harm in including it :)

### CODE REVIEW

**Comment by quinnjn** [comment]
Hi @brianguertin! Thanks for the PR!

I'm intrigued that you had to debug something in `BraintreeFragment`. Do you mind explaining what you were debugging? Also, how would knowing the stack trace have made this debugging easier?

**Comment by brianguertin** [comment]
Oh, just noticed your comment.

The reason I want to differentiate is because I prefer handle network connectivity issues in a different manner (E.g. informing the user they are offline or have no internet connectivity). I also ensure that general connectivity issues don't raise alerts in our app's bug tracking tools.

Currently, `ConfigurationException` swallows connectivity issues, so I can't detect them and handle them differently.

File: Braintree/src/main/java/com/braintreepayments/api/exceptions/ConfigurationException.java [hunk 1]
```diff
@@ -4,6 +4,9 @@
  * Error class thrown when a configuration value is invalid
  */
 public class ConfigurationException extends BraintreeException {
+    public ConfigurationException(String message, Throwable cause) {
```
Comment (lkorth) [inline Braintree/src/main/java/com/braintreepayments/api/exceptions/ConfigurationException.java]: Needs a new line before this line.

File: Braintree/src/main/java/com/braintreepayments/api/exceptions/ConfigurationException.java [hunk 2]
```diff
@@ -4,6 +4,9 @@
...
+        super(message, cause);
+    }
```
Comment (lkorth) [inline Braintree/src/main/java/com/braintreepayments/api/exceptions/ConfigurationException.java]: Needs a new line after this line.

**Comment by lkorth** [comment]
Thanks again for the PR @brianguertin! Just two whitespace changes and we'll get this merged in.

**Pull request merged**